templates/CRM/Contribute/Form/Contribution/Main.tpl

```diff
@@ -127,7 +127,7 @@
               {/if}
             {else}
               <div class="label">{$form.start_date.label}</div>
-              <div class="content">{$start_date_display|date_format}</div>
+              <div class="content">{$start_date_display|crmDate}</div>
```

Comment:
I'm not sure about this one. Without a format string these give different formats, which is probably system-dependent. On my test site I get something like `Jan  2, 2021` for date_format, and `January 2nd, 2021 12:13 PM` for crmDate.